bench-tps/src/send_batch.rs

```diff
@@ -368,9 +368,11 @@ impl<'a> FundingTransactions<'a> for FundingContainer<'a> {
     ) {
         self.make(to_fund, |(k, t)| -> (FundingSigners<'a>, Transaction) {
             let mut instructions = system_instruction::transfer_many(&k.pubkey(), t);
-            instructions.push(
-                ComputeBudgetInstruction::set_loaded_accounts_data_size_limit(data_size_limit),
-            );
+            if data_size_limit != 0 {
```

Comment:
If `data_size_limit` becomes `Option<u32>`:
```suggestion
            if let Some(data_size_limit) = data_size_limit
```